bin/fmt

```diff
@@ -28,7 +28,45 @@ if [[ -n "${CI}" ]]; then
   interactive_flag=""
 fi
 
-docker run --rm ${interactive_flag} \
-  -v "$(pwd):/code" \
-  civiform/formatter \
-  "$@"
+gitdir="$(git rev-parse --show-toplevel)"
+repotype="not-worktree"
```

Comment:
**praise:** please see styleguide for variable naming convention https://google.github.io/styleguide/shellguide.html

```suggestion
readonly GIT_DIR="$(git rev-parse --show-toplevel)"
repo_type="not-worktree"
```

Reply:
Done